File: app/src/main/java/com/hjc/reader/bean/db/History.java
package com.hjc.reader.bean.db;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;


@Entity
public class History {
    @PrimaryKey(autoGenerate = true)
    private int id;

    private String name;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @NonNull
    @Override
    public String toString() {
        return "History{" +
                "id=" + id +
                ", name='" + name + '\'' +
                '}';
    }
}

File: module-main/src/main/java/com/hjc/learn/main/model/History.kt
package com.hjc.learn.main.model

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
class History {
    @PrimaryKey(autoGenerate = true)
    var id: Int = 0

    var name = ""

    override fun toString(): String {
        return "History(id=$id, name='$name')"
    }

}